### DISCUSSION TITLE
Best way to handle OpnSense firewall logs

### DISCUSSION THREAD

**Discussion by volitank** [utmstack/UTMStack#493] [Q&A] [answered]
I know there is a pfsense integration, but I'm not sure how to enable that, or if it would work for opnsense.

The way that the logs are coming in seems to make it very difficult to do anything meaningful with them as far as visualizations.

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @volitank we do have a pfsense integration, but it isn't available in the current version, will be in the next versions. Actually has been tested only with the firewall. About opnsense, we'll take a note to review latter. Best regards

**Reply by ccesario** [reply]
Hi @c3s4rfred, currently we have migrate all of ours pfSense to OPNSense.
How is the support to integration with OPNSense ? 

I'd like to integrate with UTMStack .

regards, 
Carlos
